tests/unittests/common/unittests-constants.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2015 Philipp Rosenkranz <philipp.rosenkranz@fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#ifndef __UNITTESTS_CONSTANTS_H__
+#define __UNITTESTS_CONSTANTS_H_
+#include "embUnit/embUnit.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// Some randomly generated but still deterministic values for testing
+#define TEST_STRING4 "J&(d"
+#define TEST_STRING8 "o\\rrB/q"
+#define TEST_STRING12 "50U'HLKC3_ft"
+#define TEST_STRING16 "nvxuO*6o3C=a6g7]"
+#define TEST_STRING64 "3key9'=4\\wvAM&;.E&U/rOG:>\\C!r{K k.E[%;(}_I(D%XQk8+T>_M^G'Mm49,n:))]}"
+#define TEST_INT8 (-69)
+#define TEST_INT16 (-12229)
+#define TEST_INT32 (-1894529023L)
+#define TEST_INT64 (-4562349816927799244LL)
+#define TEST_UINT8 (95U)
+#define TEST_UINT16 (10098U)
+#define TEST_UINT32 (2831907245LU)
+#define TEST_UINT64 (13500266397057512199LLU)
```

Comment:
Would be nice if this would still be overwritable at compile time (i.e. individual ifndefs around them). Also: Travis WILL complain about doc here ;)